components/constants/webui_url_constants.h

```diff
@@ -102,6 +103,9 @@ inline constexpr char kBraveUINewTabURL[] = "chrome://newtab/";
 inline constexpr char kBraveUINewTabTakeoverURL[] =
     "chrome://new-tab-takeover/";
 
+inline constexpr char kBraveUINewTabTakeoverPageURL[] =
+    "chrome://new-tab-takeover-page/";
```

Comment:
For Android we use `chrome://new-tab-takeover` url for WebUI and 
`chrome-untrusted://new-tab-takeover` url for data source which serves NTT html files.

As I understand, this is not possible on iOS because different data sources on iOS cannot have the same host, but different schemes. That's why new url is needed.

iOS team to confirm.